setup.cfg

```diff
@@ -0,0 +1,4 @@
+[nosetests]
+tests = test/unit,
+        utils,
+        roles/openshift_master_facts/test,
```

Comment:
> consider moving test/unit/modify_yaml_tests.py -> library/test_modify_yaml.py, so that test files live close to what they test

@detiber what do you think about that idea?